### ISSUE TITLE
Dart SDK version warning for 2.0-dev

### ISSUE THREAD

**Issue by bergwerf** [Dart-Code/Dart-Code#1144]
I would say the warning about the Dart SDK being <2.0.0 should not be displayed for users using the unstable channel.

**Comment by guoranliaoran** [comment]
me too.

**Comment by bergwerf** [comment]
For me I'm fine with my OS notifying me when a new Dart version is available from the repository :)

**Comment by DanTup** [comment]
Yep, I'd recommend just setting that to false if you don't want notifications at all.

**Comment by DanTup** [comment]
Maybe there should be a button on the prompt to quickly disable it for you?

**Comment by bergwerf** [comment]
That would certainly be neat.

**Comment by DanTup** [comment]
Ok, I've renamed this issue to be for that. It'll just set the global setting for you so you can easily revert it if required.

**Comment by DanTup** [comment]
Please try the fix out in v2.18.0 Beta 1:

https://github.com/Dart-Code/Dart-Code/releases/tag/v2.18.0-beta.1
